From a8a1081ec05634e667d6670ab7c128d536cc60a9 Mon Sep 17 00:00:00 2001
From: Nicolas Dechesne <nicolas.dechesne@linaro.org>
Date: Thu, 1 Jun 2017 11:59:42 +0200
Subject: [PATCH] util/rand_xor: add missing include statements

Fixes for:

src/util/rand_xor.c:60:13: error: implicit declaration of function 'open' [-Werror=implicit-function-declaration]
    int fd = open("/dev/urandom", O_RDONLY);
             ^~~~
src/util/rand_xor.c:60:34: error: 'O_RDONLY' undeclared (first use in this function)
    int fd = open("/dev/urandom", O_RDONLY);
                                  ^~~~~~~~

Upstream-Status: Submitted [ https://lists.freedesktop.org/archives/mesa-dev/2017-June/157542.html ]
Signed-off-by: Nicolas Dechesne <nicolas.dechesne@linaro.org>
---
 src/util/rand_xor.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/util/rand_xor.c b/src/util/rand_xor.c
index de05fa64b3..de04bbc284 100644
--- a/src/util/rand_xor.c
+++ b/src/util/rand_xor.c
@@ -25,6 +25,7 @@
 #if defined(__linux__)
 #include <sys/file.h>
 #include <unistd.h>
+#include <fcntl.h>
 #else
 #include <time.h>
 #endif
-- 
2.13.0

